internal/restic/node_windows.go

```diff
@@ -32,6 +32,9 @@ var (
 	modAdvapi32     = syscall.NewLazyDLL("advapi32.dll")
 	procEncryptFile = modAdvapi32.NewProc("EncryptFileW")
 	procDecryptFile = modAdvapi32.NewProc("DecryptFileW")
+
+	// eaUnsupportedVolumesMap is a map of volumes that do not support extended attributes.
+	eaUnsupportedVolumesMap = map[string]bool{}
```

Comment:
I'm really no fan of introducing additional global state as it usually leads to problems sooner or later. However, until I've moved the NodeFromFileInfo and related code into the `internal/fs` package, I don't see a good  way to avoid it. Either way this map MUST be synchronized.

Reply:
I agree. We can revisit later
Fixed using sync.Map